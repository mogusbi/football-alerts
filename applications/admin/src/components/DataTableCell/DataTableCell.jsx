import TableCell from '@material-ui/core/TableCell';
import React, {memo} from 'react';

function DataTableCell (props) {
  return (
    <TableCell {...props} />
  );
}

export default memo(DataTableCell);
