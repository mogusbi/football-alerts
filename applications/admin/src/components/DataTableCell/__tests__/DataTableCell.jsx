import {mount} from 'enzyme';
import React from 'react';
import DataTableCell from '../DataTableCell';

describe('DataTableCell', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(
      <table>
        <tbody>
          <tr>
            <DataTableCell />
          </tr>
        </tbody>
      </table>
    );
  });

  it('should render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
