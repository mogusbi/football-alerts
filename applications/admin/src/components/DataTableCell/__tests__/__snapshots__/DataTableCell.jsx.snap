// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTableCell should render component 1`] = `
<table>
  <tbody>
    <tr>
      <DataTableCell>
        <WithStyles(TableCell)>
          <TableCell
            align="inherit"
            classes={
              Object {
                "alignCenter": "MuiTableCell-alignCenter-10",
                "alignJustify": "MuiTableCell-alignJustify-12",
                "alignLeft": "MuiTableCell-alignLeft-9",
                "alignRight": "MuiTableCell-alignRight-11",
                "body": "MuiTableCell-body-3",
                "footer": "MuiTableCell-footer-4",
                "head": "MuiTableCell-head-2",
                "numeric": "MuiTableCell-numeric-5",
                "paddingCheckbox": "MuiTableCell-paddingCheckbox-7",
                "paddingDense": "MuiTableCell-paddingDense-6",
                "paddingNone": "MuiTableCell-paddingNone-8",
                "root": "MuiTableCell-root-1",
              }
            }
          >
            <td
              aria-sort={null}
              className="MuiTableCell-root-1"
            />
          </TableCell>
        </WithStyles(TableCell)>
      </DataTableCell>
    </tr>
  </tbody>
</table>
`;
